packages/expo-notifications/src/setNotificationCategoryAsync.ts

```diff
@@ -23,7 +23,7 @@ export default async function setNotificationCategoryAsync(
   identifier: string,
   actions: NotificationAction[],
   options?: NotificationCategoryOptions
-): Promise<NotificationCategory> {
+): Promise<NotificationCategory | null> {
```

Comment:
This is a public facing type change. I'm unsure of the consequences of it.

Reply:
Interesting. I think it's probably fine but it could definitely cause a `tsc` error in people's apps (which is okay, we'll just want to make sure it's in the changelog).

Reply:
Yeah, it seems fine for this simple change, I wouldn't consider it as breaking 👍